extensions/amp-script/0.1/amp-script.js

```diff
@@ -789,5 +786,12 @@ export class SanitizerImpl {
 
 AMP.extension(TAG, '0.1', function(AMP) {
   AMP.registerServiceForDoc(TAG, AmpScriptService);
+  AMP.registerServiceForDoc('purifier-inplace', ampdoc => {
```

Comment:
please use normal function instead of arrow func, i believe this will be `new`d up at some point. did the linter catch this?